src/renderers/native/ReactNativeFiberErrorDialog.js

```diff
@@ -43,6 +48,7 @@ function ReactNativeFiberErrorDialog(capturedError: CapturedError): boolean {
   }
 
   const newError = new errorType(errorMessage);
```

Comment:
Like mention above, this constructor may have different signatures than accepting a message for some error types.